{
  "uid" : "d8a0fe57f151745c",
  "name" : "Verify value of the title of the tab",
  "fullName" : "tests.specialists_page_test.TestSpecialistPageText#test_sp_02_01_verify_tab_title",
  "historyId" : "f48b9972ca3305ef3141feb4c3e1a46f",
  "time" : {
    "start" : 1728784204730,
    "stop" : 1728784204730,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55c8643d402a <unknown>\n#1 0x55c8640ba5e0 <unknown>\n#2 0x55c8640b27eb <unknown>\n#3 0x55c8640a2a09 <unknown>\n#4 0x55c8640a4744 <unknown>\n#5 0x55c8640a2cbd <unknown>\n#6 0x55c8640a252f <unknown>\n#7 0x55c8640a2422 <unknown>\n#8 0x55c8640a0493 <unknown>\n#9 0x55c8640a0ada <unknown>\n#10 0x55c8640bcf41 <unknown>\n#11 0x55c86414e8c5 <unknown>\n#12 0x55c86412eb22 <unknown>\n#13 0x55c86414dd7d <unknown>\n#14 0x55c86412e8c3 <unknown>\n#15 0x55c8640fc6b3 <unknown>\n#16 0x55c8640fd68e <unknown>\n#17 0x55c86439ea2b <unknown>\n#18 0x55c8643a29b1 <unknown>\n#19 0x55c86438b225 <unknown>\n#20 0x55c8643a3532 <unknown>\n#21 0x55c86437038f <unknown>\n#22 0x55c8643c2f28 <unknown>\n#23 0x55c8643c30f3 <unknown>\n#24 0x55c8643d2e7c <unknown>\n#25 0x7fb72e894ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"b433118d9d0d7640d3f30104c8662042\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7faf1f689dc0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x55c8643c30f3 \\\\u003Cunknown>\\\\n#24 0x55c8643d2e7c \\\\u003Cunknown>\\\\n#25 0x7fb72e894ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x55c8643d402a <unknown>\nE       #1 0x55c8640ba5e0 <unknown>\nE       #2 0x55c8640b27eb <unknown>\nE       #3 0x55c8640a2a09 <unknown>\nE       #4 0x55c8640a4744 <unknown>\nE       #5 0x55c8640a2cbd <unknown>\nE       #6 0x55c8640a252f <unknown>\nE       #7 0x55c8640a2422 <unknown>\nE       #8 0x55c8640a0493 <unknown>\nE       #9 0x55c8640a0ada <unknown>\nE       #10 0x55c8640bcf41 <unknown>\nE       #11 0x55c86414e8c5 <unknown>\nE       #12 0x55c86412eb22 <unknown>\nE       #13 0x55c86414dd7d <unknown>\nE       #14 0x55c86412e8c3 <unknown>\nE       #15 0x55c8640fc6b3 <unknown>\nE       #16 0x55c8640fd68e <unknown>\nE       #17 0x55c86439ea2b <unknown>\nE       #18 0x55c8643a29b1 <unknown>\nE       #19 0x55c86438b225 <unknown>\nE       #20 0x55c8643a3532 <unknown>\nE       #21 0x55c86437038f <unknown>\nE       #22 0x55c8643c2f28 <unknown>\nE       #23 0x55c8643c30f3 <unknown>\nE       #24 0x55c8643d2e7c <unknown>\nE       #25 0x7fb72e894ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728784074935,
      "stop" : 1728784074936,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728784205503,
      "stop" : 1728784205926,
      "duration" : 423
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55c8643d402a <unknown>\n#1 0x55c8640ba5e0 <unknown>\n#2 0x55c8640b27eb <unknown>\n#3 0x55c8640a2a09 <unknown>\n#4 0x55c8640a4744 <unknown>\n#5 0x55c8640a2cbd <unknown>\n#6 0x55c8640a252f <unknown>\n#7 0x55c8640a2422 <unknown>\n#8 0x55c8640a0493 <unknown>\n#9 0x55c8640a0ada <unknown>\n#10 0x55c8640bcf41 <unknown>\n#11 0x55c86414e8c5 <unknown>\n#12 0x55c86412eb22 <unknown>\n#13 0x55c86414dd7d <unknown>\n#14 0x55c86412e8c3 <unknown>\n#15 0x55c8640fc6b3 <unknown>\n#16 0x55c8640fd68e <unknown>\n#17 0x55c86439ea2b <unknown>\n#18 0x55c8643a29b1 <unknown>\n#19 0x55c86438b225 <unknown>\n#20 0x55c8643a3532 <unknown>\n#21 0x55c86437038f <unknown>\n#22 0x55c8643c2f28 <unknown>\n#23 0x55c8643c30f3 <unknown>\n#24 0x55c8643d2e7c <unknown>\n#25 0x7fb72e894ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728784205503,
        "stop" : 1728784205926,
        "duration" : 423
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55c8643d402a <unknown>\n#1 0x55c8640ba5e0 <unknown>\n#2 0x55c8640b27eb <unknown>\n#3 0x55c8640a2a09 <unknown>\n#4 0x55c8640a4744 <unknown>\n#5 0x55c8640a2cbd <unknown>\n#6 0x55c8640a252f <unknown>\n#7 0x55c8640a2422 <unknown>\n#8 0x55c8640a0493 <unknown>\n#9 0x55c8640a0ada <unknown>\n#10 0x55c8640bcf41 <unknown>\n#11 0x55c86414e8c5 <unknown>\n#12 0x55c86412eb22 <unknown>\n#13 0x55c86414dd7d <unknown>\n#14 0x55c86412e8c3 <unknown>\n#15 0x55c8640fc6b3 <unknown>\n#16 0x55c8640fd68e <unknown>\n#17 0x55c86439ea2b <unknown>\n#18 0x55c8643a29b1 <unknown>\n#19 0x55c86438b225 <unknown>\n#20 0x55c8643a3532 <unknown>\n#21 0x55c86437038f <unknown>\n#22 0x55c8643c2f28 <unknown>\n#23 0x55c8643c30f3 <unknown>\n#24 0x55c8643d2e7c <unknown>\n#25 0x7fb72e894ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"b433118d9d0d7640d3f30104c8662042\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1728784204731,
      "stop" : 1728784205502,
      "duration" : 771
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728784205961,
      "stop" : 1728784206023,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Specialists Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "specialists_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestSpecialistsPage"
  }, {
    "name" : "host",
    "value" : "fv-az891-551"
  }, {
    "name" : "thread",
    "value" : "2367-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.specialists_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 1,
        "broken" : 3,
        "skipped" : 0,
        "passed" : 27,
        "unknown" : 0,
        "total" : 31
      },
      "items" : [ {
        "uid" : "8b341abfe03f9b45",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/60//#testresult/8b341abfe03f9b45",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x557b46c0402a <unknown>\n#1 0x557b468ea5e0 <unknown>\n#2 0x557b468e27eb <unknown>\n#3 0x557b468d2a09 <unknown>\n#4 0x557b468d4744 <unknown>\n#5 0x557b468d2cbd <unknown>\n#6 0x557b468d252f <unknown>\n#7 0x557b468d2422 <unknown>\n#8 0x557b468d0493 <unknown>\n#9 0x557b468d0ada <unknown>\n#10 0x557b468ecf41 <unknown>\n#11 0x557b4697e8c5 <unknown>\n#12 0x557b4695eb22 <unknown>\n#13 0x557b4697dd7d <unknown>\n#14 0x557b4695e8c3 <unknown>\n#15 0x557b4692c6b3 <unknown>\n#16 0x557b4692d68e <unknown>\n#17 0x557b46bcea2b <unknown>\n#18 0x557b46bd29b1 <unknown>\n#19 0x557b46bbb225 <unknown>\n#20 0x557b46bd3532 <unknown>\n#21 0x557b46ba038f <unknown>\n#22 0x557b46bf2f28 <unknown>\n#23 0x557b46bf30f3 <unknown>\n#24 0x557b46c02e7c <unknown>\n#25 0x7f5c43894ac3 <unknown>",
        "time" : {
          "start" : 1728697318632,
          "stop" : 1728697318632,
          "duration" : 0
        }
      }, {
        "uid" : "7346e860c65bece2",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/59//#testresult/7346e860c65bece2",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x559aaac4402a <unknown>\n#1 0x559aaa92a5e0 <unknown>\n#2 0x559aaa9227eb <unknown>\n#3 0x559aaa912a09 <unknown>\n#4 0x559aaa914744 <unknown>\n#5 0x559aaa912cbd <unknown>\n#6 0x559aaa91252f <unknown>\n#7 0x559aaa912422 <unknown>\n#8 0x559aaa910493 <unknown>\n#9 0x559aaa910ada <unknown>\n#10 0x559aaa92cf41 <unknown>\n#11 0x559aaa9be8c5 <unknown>\n#12 0x559aaa99eb22 <unknown>\n#13 0x559aaa9bdd7d <unknown>\n#14 0x559aaa99e8c3 <unknown>\n#15 0x559aaa96c6b3 <unknown>\n#16 0x559aaa96d68e <unknown>\n#17 0x559aaac0ea2b <unknown>\n#18 0x559aaac129b1 <unknown>\n#19 0x559aaabfb225 <unknown>\n#20 0x559aaac13532 <unknown>\n#21 0x559aaabe038f <unknown>\n#22 0x559aaac32f28 <unknown>\n#23 0x559aaac330f3 <unknown>\n#24 0x559aaac42e7c <unknown>\n#25 0x7f942fe94ac3 <unknown>",
        "time" : {
          "start" : 1728611009863,
          "stop" : 1728611009863,
          "duration" : 0
        }
      }, {
        "uid" : "5176d376d17886da",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/58//#testresult/5176d376d17886da",
        "status" : "passed",
        "time" : {
          "start" : 1728524988289,
          "stop" : 1728524988302,
          "duration" : 13
        }
      }, {
        "uid" : "45c267cc19fe1f8b",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/57//#testresult/45c267cc19fe1f8b",
        "status" : "passed",
        "time" : {
          "start" : 1728438660409,
          "stop" : 1728438660421,
          "duration" : 12
        }
      }, {
        "uid" : "a17ee2f8b415e1c0",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/56//#testresult/a17ee2f8b415e1c0",
        "status" : "passed",
        "time" : {
          "start" : 1728352245393,
          "stop" : 1728352245406,
          "duration" : 13
        }
      }, {
        "uid" : "ca0e81fd0ab23d79",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/55//#testresult/ca0e81fd0ab23d79",
        "status" : "failed",
        "statusDetails" : "AssertionError: The title value of the tab mismatches the valid value\nassert 'BrainUp' in ['Специалисты | BrainUp', 'Specialists | BrainUp']\n +  where ['Специалисты | BrainUp', 'Specialists | BrainUp'] = SpecialistsPageData.tab_title",
        "time" : {
          "start" : 1728266694385,
          "stop" : 1728266694398,
          "duration" : 13
        }
      }, {
        "uid" : "6b083705e39c066a",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/54//#testresult/6b083705e39c066a",
        "status" : "passed",
        "time" : {
          "start" : 1728179788576,
          "stop" : 1728179788588,
          "duration" : 12
        }
      }, {
        "uid" : "e310e81cca336259",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/53//#testresult/e310e81cca336259",
        "status" : "passed",
        "time" : {
          "start" : 1728093026635,
          "stop" : 1728093026648,
          "duration" : 13
        }
      }, {
        "uid" : "248b6bba1986f090",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/52//#testresult/248b6bba1986f090",
        "status" : "passed",
        "time" : {
          "start" : 1728006729784,
          "stop" : 1728006729797,
          "duration" : 13
        }
      }, {
        "uid" : "8d21aefde3eebd6f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/51//#testresult/8d21aefde3eebd6f",
        "status" : "passed",
        "time" : {
          "start" : 1727920374317,
          "stop" : 1727920374331,
          "duration" : 14
        }
      }, {
        "uid" : "ec29eeabdae6366e",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/50//#testresult/ec29eeabdae6366e",
        "status" : "passed",
        "time" : {
          "start" : 1727833798263,
          "stop" : 1727833798276,
          "duration" : 13
        }
      }, {
        "uid" : "3a9a87b26ae0827f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/49//#testresult/3a9a87b26ae0827f",
        "status" : "passed",
        "time" : {
          "start" : 1727747922966,
          "stop" : 1727747922980,
          "duration" : 14
        }
      }, {
        "uid" : "714d513b03dd0907",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/48//#testresult/714d513b03dd0907",
        "status" : "passed",
        "time" : {
          "start" : 1727661204618,
          "stop" : 1727661204631,
          "duration" : 13
        }
      }, {
        "uid" : "8326839b76814c89",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/47//#testresult/8326839b76814c89",
        "status" : "passed",
        "time" : {
          "start" : 1727575025941,
          "stop" : 1727575025953,
          "duration" : 12
        }
      }, {
        "uid" : "963112e4182746e9",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/46//#testresult/963112e4182746e9",
        "status" : "passed",
        "time" : {
          "start" : 1727488252407,
          "stop" : 1727488252427,
          "duration" : 20
        }
      }, {
        "uid" : "7a7f2c38f4537260",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/45//#testresult/7a7f2c38f4537260",
        "status" : "passed",
        "time" : {
          "start" : 1727401996372,
          "stop" : 1727401996384,
          "duration" : 12
        }
      }, {
        "uid" : "87d20e1044632912",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/44//#testresult/87d20e1044632912",
        "status" : "passed",
        "time" : {
          "start" : 1727315470588,
          "stop" : 1727315470601,
          "duration" : 13
        }
      }, {
        "uid" : "fa5007edd95de1d2",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/43//#testresult/fa5007edd95de1d2",
        "status" : "passed",
        "time" : {
          "start" : 1727229120300,
          "stop" : 1727229120320,
          "duration" : 20
        }
      }, {
        "uid" : "da532bd6270e24e0",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/42//#testresult/da532bd6270e24e0",
        "status" : "passed",
        "time" : {
          "start" : 1727142651613,
          "stop" : 1727142651626,
          "duration" : 13
        }
      }, {
        "uid" : "e9a33d247fc6eea7",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/41//#testresult/e9a33d247fc6eea7",
        "status" : "passed",
        "time" : {
          "start" : 1727056172295,
          "stop" : 1727056172307,
          "duration" : 12
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "d8a0fe57f151745c.json",
  "parameterValues" : [ ]
}